content/guides/trash/index.md

```diff
@@ -0,0 +1,17 @@
+---
+rank: 50
+related_endpoints: []
+related_guides: []
+required_guides: []
+related_resources:
+  - folders
+  - files
+  - web-links
+---
+
+# Trash
+
+Box uses a two-stage process to remove or "trash" files, folders, or web links
+before permanently deleting them. By default, these items can be restored
+once trashed within a 30 day period before they are purged. The purge window
+can be modified by administrators of Business or Enterprise accounts.
```

Comment:
This seems like important info. Maybe can we put this under a H2 header? And maybe add alink to the community article on trash?